### ISSUE TITLE
Horizontal line noise in output spectrogram of speech

### ISSUE THREAD

**Issue by chazo1994** [NVIDIA/tacotron2#271]
I haved trained a end to end speech synthesis system with 16hours of 16000hz audio. I use nvidia-tacotron2 with my hparams like below:
ignore_layers=[''],
text_cleaners=['basic_cleaners'],
fp16_run=True,
batch_size=64,
sampling_rate=16000,
All the remaining parameters I set default. After 500 epoch I got output speech with horizontal line noise. The spectrogram of output speech like this:
![image](https://user-images.githubusercontent.com/17020334/66654966-7c228580-ec65-11e9-9c2b-6923ca783b96.png)

And the alignment of our model like this:
![image](https://user-images.githubusercontent.com/17020334/66655064-a5431600-ec65-11e9-9da2-90bb89beedd3.png)

Please help me explain the problems and how to fix it.

**Comment by rafaelvalle** [comment]
That horizontal line waveglow's model bias. You can remove the bias by fine-tuning the model or by increasing the strength of the denoiser.

**Comment by chazo1994** [comment]
@rafaelvalle  
I trained a waveglow model with 10h of recording, and I used your pretrain model for warmup. But after 400 epoch, the vertical line noise still exist in some output test sample. I change strengh of denoise form 0.01 to 0.1 (step 0.1), with strengh 0.1 the noise is gone. But the quality of voice is so bad. So, do have any sugestion to deal with this problem, or have some others way to fine tuning the model. Thank you so much.
